core/src/tree/TreeOptions.cpp

```diff
@@ -25,14 +25,16 @@ TreeOptions::TreeOptions(uint mtry,
                          double honesty_fraction,
                          bool honesty_prune_leaves,
                          double alpha,
-                         double imbalance_penalty):
+                         double imbalance_penalty,
+                         uint block_length):
```

Comment:
need a default argument here, in case mbb=false

Reply:
done